frontend/components/MainPageV1/Home/Overview/Wallet.tsx

```diff
@@ -1,22 +1,33 @@
-import { Button, Flex, Typography } from 'antd';
-import { isNumber } from 'lodash';
+import { Button, Flex, Image, Typography } from 'antd';
+import styled from 'styled-components';
 
 import { CardFlex } from '@/components/ui/CardFlex';
-import { NA } from '@/constants/symbols';
+import { COLOR, NA, TokenSymbolConfigMap, TokenSymbolMap } from '@/constants';
 import { Pages } from '@/enums/Pages';
 import { useSharedContext } from '@/hooks';
+import { useAvailableAgentAssets } from '@/hooks/useAvailableAgentAssets';
 import { usePageState } from '@/hooks/usePageState';
 
 const { Text, Title } = Typography;
 
-// TODO
+const TokenWrapper = styled(Flex)`
+  display: inline-flex;
+  width: fit-content;
+  padding: 4px 10px 4px 6px;
+  border-radius: 8px;
+  border: 1px solid ${COLOR.GRAY_3};
+`;
+
 /**
  * To display wallet overview on the main page.
  */
 export const Wallet = () => {
   const { isMainOlasBalanceLoading } = useSharedContext();
   const { goto } = usePageState();
-  const aggregatedBalanceInUsdTODO = null;
+  const availableAssets = useAvailableAgentAssets();
+  const availableAssetsExceptOlas = availableAssets
+    .filter(({ symbol }) => symbol !== TokenSymbolMap.OLAS)
+    .filter(({ amount }) => amount > 0);
```

Comment:
why two filters if can be one? `symbol !== TokenSymbolMap.OLAS && amount > 0`

Reply:
just a little cleaner 😅 , but i get it, loop would be running twice, will move